fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -283,6 +287,24 @@ class DDTeamCollectionImpl {
 				}
 			}
 
+			// When relocating shards, DD would evaluate the CPU & AvailabeSpace of sourceTeam. If
+			// both of metrics are below pivot values, we would return the source team to dataDistributionRelocator
+			if (req.teamSelect.isForRelocateShard() && SERVER_KNOBS->DD_REEVALUATION_ENABLED) {
+				auto sourceTeam = self->evaluateSourceTeam(req.src);
+				if (sourceTeam.present()) {
+					CODE_PROBE(true, "dd re-evaluation triggered");
+					TraceEvent(SevInfo, "GetTeamReturnSourceTeam", self->distributorId)
```

Comment:
It seems like this trace event could be too noisy. Are there other traces we print for every relocation?

Reply:
Actually, we might have enough traces already.

1. Each `dataDistributionRelocator` would print a `RelocateShardHasDestination` which has all the info of a single request.

2. `getTeam` will print `TeamEligibilityCount`, which includes the pivot values.

I will change the severity to `SevDebug` here;  enable `TeamEligibilityCount` to print strict pivot values as well.